infra/api.py

```diff
@@ -121,6 +123,13 @@ def comment():
     data = json.load(bottle.request.body)
     DATA.chapter_comment(**data)
 
+@bottle.post("/api/quiz_telemetry", method=['OPTIONS', 'POST'])
```

Comment:
Do quiz questions have some kind of stable ID? If we change the questions how do we track 1) which version the answers are for, and 2) what question it was?

Reply:
I don't think there's a versioning system—that's a really good idea though. I can see what I can put together this week.